cmake/test_model_zoo.cmake

```
@@ -108,7 +108,7 @@ if(ENABLE_TESTS)
 
         message(STATUS "pip version is ${pip3_version}")
         set(args --quiet)
-        if(pip3_version VERSION_GREATER 20.2.2)
+        if(pip3_version VERSION_GREATER 20.2.2 AND pip3_version VERSION_LESS 21.0.0)
```

Citing from: https://pip.pypa.io/en/stable/user_guide/#deprecation-timeline
>pip 20.3: pip defaults to the new resolver in Python 3 environments

Shouldn't it be `20.3` instead?
```suggestion
        if(pip3_version VERSION_GREATER 20.2.2 AND pip3_version VERSION_LESS 20.3.0)
```